packages/apps/src/Apps.tsx

```diff
@@ -113,33 +113,43 @@ export default React.memo(styled(Apps)`
   flex-direction: row;
   height: 100vh;
 
+  * {
+    font-family: 'Nunito', sans-serif !important;
```

Comment:
This is problematic since it applies to the UI, but not to the modals. So we have a mixture of fonts, see for instance -

![image](https://user-images.githubusercontent.com/1424473/90228909-55018b00-de17-11ea-972d-4dcf6b4c736c.png)